### PR TITLE
tlg0013_tlg024

### PR DESCRIPTION
**Pull request by AlisonBabeu**
Final files for Homeric Hymns. Same changes as previous.

### CODE REVIEW

File: data/tlg0013/tlg025/__cts__.xml [hunk 1]
```diff
@@ -5,19 +5,19 @@
       
    <ti:translation projid="greekLit:perseus-eng2" urn="urn:cts:greekLit:tlg0013.tlg025.perseus-eng2" xml:lang="eng" workUrn="urn:cts:greekLit:tlg0013.tlg025">
         
-       <ti:label xml:lang="eng">Hymn 25 to the Muses and Apollo, Hesiod The Homeric hymns. And Homerica</ti:label>
+       <ti:label xml:lang="eng">Hymn 25 to the Muses and Apollo</ti:label>
         
-       <ti:description xml:lang="eng">Hymni Homerici, creator; Hesiod, creator; Homer, creator; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, editor; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, translator</ti:description>
+       <ti:description xml:lang="eng">Anonymous. Hesiod, The Homeric Hymns and Homerica. Evelyn-White, Hugh G., translatr. London: William Heinmann; New York: The Macmillan Co., 1914.</ti:description>
```
Comment (lcerrato) [inline data/tlg0013/tlg025/__cts__.xml:10]: typo in translator

File: data/tlg0013/tlg029/__cts__.xml [hunk 2]
```diff
@@ -5,19 +5,19 @@
...
       
    </ti:translation>
     
     <ti:edition projid="greekLit:perseus-grc2" urn="urn:cts:greekLit:tlg0013.tlg029.perseus-grc2" workUrn="urn:cts:greekLit:tlg0013.tlg029" xml:lang="grc">
         
-        <ti:label xml:lang="eng">Hymn 29 to Hestia, Hesiod The Homeric hymns. And Homerica</ti:label>
+        <ti:label xml:lang="eng">Hymn 29 to Hestia</ti:label>
         
-        <ti:description xml:lang="eng">Hymni Homerici, creator; Hesiod, creator; Homer, creator; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, translator</ti:description>
+        <ti:description xml:lang="eng">Anonymous. Hesiod, The Homeric Hymns and Homerica. Evelyn-White, Hugh G., translator. London: William Heinmann; New York: The Macmillan Co., 1914.</ti:description>
```
Comment (lcerrato) [inline data/tlg0013/tlg029/__cts__.xml:20]: editor

File: data/tlg0013/tlg032/__cts__.xml [hunk 3]
```diff
@@ -5,19 +5,19 @@
...
       
    </ti:translation>
     
     <ti:edition projid="greekLit:perseus-grc2" urn="urn:cts:greekLit:tlg0013.tlg032.perseus-grc2" workUrn="urn:cts:greekLit:tlg0013.tlg032" xml:lang="grc">
         
-        <ti:label xml:lang="eng">Hymn 32 to Selene, Hesiod The Homeric hymns. And Homerica</ti:label>
+        <ti:label xml:lang="eng">Hymn 32 to Selene</ti:label>
         
-        <ti:description xml:lang="eng">Hymni Homerici, creator; Hesiod, creator; Homer, creator; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, translator</ti:description>
+        <ti:description xml:lang="eng">Anonymous. Hesiod, The Homeric Hymns and Homerica. Evelyn-White, Hugh G., translator. London: William Heinmann; New York: The Macmillan Co., 1914.</ti:description>
```
Comment (lcerrato) [inline data/tlg0013/tlg032/__cts__.xml:20]: editor

File: data/tlg0013/tlg033/__cts__.xml [hunk 4]
```diff
@@ -5,19 +5,19 @@
...
       
    </ti:translation>
     
     <ti:edition projid="greekLit:perseus-grc2" urn="urn:cts:greekLit:tlg0013.tlg033.perseus-grc2" workUrn="urn:cts:greekLit:tlg0013.tlg033" xml:lang="grc">
         
-        <ti:label xml:lang="eng">Hymn 33 to the Dioscuri, Hesiod The Homeric hymns. And Homerica</ti:label>
+        <ti:label xml:lang="eng">Hymn 33 to the Dioscuri</ti:label>
         
-        <ti:description xml:lang="eng">Hymni Homerici, creator; Hesiod, creator; Homer, creator; Evelyn-White, Hugh G. (Hugh Gerard), d. 1924, translator</ti:description>
+        <ti:description xml:lang="eng">Anonymous. Hesiod, The Homeric Hymns and Homerica. Evelyn-White, Hugh G., translator. London: William Heinmann; New York: The Macmillan Co., 1914.</ti:description>
```
Comment (lcerrato) [inline data/tlg0013/tlg033/__cts__.xml:20]: editor

**Review by lcerrato — COMMENTED** [review]
Looks like a few editor/translator errors in the cts files

File: data/tlg0013/tlg025/__cts__.xml (hunk 1 above)
Comment (AlisonBabeu) [reply data/tlg0013/tlg025/__cts__.xml:10]: hi @lcerrato aw, got sloppy at the end! thanks for catching!

**Review by lcerrato — APPROVED** [review]

**Pull request merged**